NitroxServer/ConsoleCommands/AutoBackupCommand.cs

```diff
@@ -0,0 +1,42 @@
+using NitroxModel.DataStructures.GameLogic;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.ConsoleCommands.Abstract.Type;
+using NitroxServer.Serialization;
+using NitroxServer.Serialization.World;
+using System.IO;
+
+namespace NitroxServer.ConsoleCommands
+{
+    internal class AutoBackupCommand : Command
```

Comment:
I don't see any reason to have a dynamic autobackup enable/disable command, if the server owner wants to disable the backup behavior, he can change the config and restart the server